### PR TITLE
Fixes for ipxe

### PR DESCRIPTION
**Pull request by giggsoff**
Removal of predefined ipxe template in favor of bundled with EVE. Improvement of docs.
Add ability to skip registry test (EDEN_TEST_REGISTRY), it will not work in case of remote EVE connected via public IPs.

### CODE REVIEW

File: cmd/edenSetup.go [hunk 1]
```diff
@@ -298,36 +296,30 @@ var setupCmd = &cobra.Command{
 						EServerPort: eServerPort,
 					}
 					files := []string{
-						"kernel", "initrd.img", "initrd.bits",
+						"kernel", "initrd.img", "initrd.bits", "installer.img", "rootfs.img",
```
Comment (rvs) [inline cmd/edenSetup.go:299]: Is there any way to NOT have this explicit list and just serve up whatever comes out of `docker run lfedge/eve installer_net | tar xvf -` ?

**Review by rvs — APPROVED** [review]
LGTM -- but would like to make it less brittle when it comes to knowing exactly what files to serve up -- see my comment inline.

File: cmd/edenSetup.go (hunk 1 above)
Comment (giggsoff) [reply cmd/edenSetup.go:299]: Good point, thank you!

Comment (giggsoff) [reply cmd/edenSetup.go]: Done

**Review by rvs — APPROVED** [review]
LGTM

**Pull request merged**